nixos/modules/services/networking/twingate.nix

```diff
@@ -0,0 +1,29 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.twingate;
+
+in {
+
+  options.services.twingate = {
+    enable = mkEnableOption "Twingate Client daemon";
+  };
+
+  config = mkIf cfg.enable {
+
+    networking.firewall.checkReversePath = lib.mkDefault false;
+
+    environment.systemPackages = [ pkgs.twingate ]; # for the CLI
+    systemd.packages = [ pkgs.twingate ];
+
+    system.activationScripts.twingate = stringAfter ["etc"] ''
+      mkdir -p '/etc/twingate'
+
+      cp -r -n ${pkgs.twingate}/etc/twingate/. /etc/twingate/
```

Comment:
Please move this line into `systemd.services.twingate.preStart` instead.

Reply:
Such change will break app logic

Reply:
Can you please explain?

Reply:
@aanderse , It seems that I mixed up something earlier. I've checked one more time and confirm that your suggestion is ok, applied.